src/esockd_acceptor.erl

```diff
@@ -134,9 +134,12 @@ handle_event(internal, begin_waiting, waiting, State = #state{lsock = LSock}) ->
             Reason =:= enfile
         ->
             {next_state, suspending, State, {state_timeout, 1000, begin_waiting}};
+        {error, econnaborted} ->
+            {next_state, waiting, State, {next_event, internal, begin_waiting}};
         {error, closed} ->
             {stop, normal, State};
         {error, Reason} ->
+            error_logger:error_msg("~p async_accept error: ~p", [?MODULE, Reason]),
```

Comment:
close the socket before stop ?
so the listener process will for sure get an EXIT signal

Reply:
just record after discussion:
We don't need to close the listen port here to avoid spreading the error to the other acceptors (where other acceptors will get closed error) since we don't know if the error is from the Listen port or some other unknown posix errors.
AND we have a listener process that handles the EXIT signal from the listen port.
AND we trust OTP will trigger port signals if listen port is unusable, unacceptable. 
AND we checked that `inet_tcp:accept` does not close the port when it gets an error, thus we follow it.